esmvaltool/recipes/recipe_energy_budget.yml

```diff
@@ -0,0 +1,130 @@
+#ESMValTool
+---
+documentation:
+  description: |
+    Recipe to compute energy budgets in global, land and ocean domains.
+    The path to the Trenberth Diagram template needs to be provided to
+    generate the plot. If left empty, the output will be a netCDF file per
+    model containing the budget variables.
+
+  authors:
+    - vanniere_benoit
+
+  maintainer:
+    - loosveldt-tomas_saskia
+
+  projects:
+    - primavera
+
+datasets:
+  - {dataset: CMCC-CM,  project: CMIP5, mip: Amon, exp: historical,  ensemble: r1i1p1,  start_year: 2000,  end_year: 2002}
+  - {dataset: CMCC-CMS,  project: CMIP5, mip: Amon, exp: historical,  ensemble: r1i1p1,  start_year: 2000,  end_year: 2002}
+#  - {dataset: HadGEM3-GC31-LM,  project: CMIP6, mip: Amon, exp: highresSST-present,  ensemble: r1i1p1f1, grid: gn,  start_year: 2000,  end_year: 2002}
```

Comment:
Why are there commented out datasets here?

Reply:
I was trying out if everything worked with CMIP6 data. Let me update this and #1285 recipes to include the datasets relevant to the project